contrib/kind.sh

```diff
@@ -359,6 +359,16 @@ fi
 kind create cluster --name ${KIND_CLUSTER_NAME} --kubeconfig ${KUBECONFIG} --image kindest/node:${K8S_VERSION} --config=${KIND_CONFIG_LCL}
 cat ${KUBECONFIG}
 
+# docker disables IPv6 globally inside containers but in the eth0 interface
+# kind enables IPv6 on the containers ONLY for dual-stack and IPv6 deployments
+# we may need to enable IPv6 for IPv4 deployments to test the migration from
+# single to dual stack. Since this is harmless we enable IPv6 always.
```

Comment:
I think there's supposed to be a period at the end of both the first and second lines? (At first I thought it was saying "docker disables IPv6 globally, but kind enables it on eth0" or something like that.)

(Changing "but" to "except" in the first line would help too.)

(The commit message for this commit is similarly confusing)

Reply:
There is another PR for this here:
https://github.com/ovn-org/ovn-kubernetes/pull/2043/files

should we move to discussing in that PR?

Reply:
https://github.com/ovn-org/ovn-kubernetes/pull/2043/files#r576988850

Reply:
moved it to this PR and reworded (hopefully is more clear now :sweat_smile: )
https://github.com/ovn-org/ovn-kubernetes/pull/2043